CMakeLists.txt

```diff
@@ -44,7 +44,18 @@ if(BUILD_MCNP5 OR BUILD_MCNP6)
   enable_language(Fortran)
 endif()
 
-find_package(MOAB REQUIRED)
+# Dealing with MOAB
+if (PULL_INSTALL_MOAB)
```

Comment:
I think we need to declare this variable somewhere to define it's default

Reply:
it is already in the `DAGMC_macros.cmake`

Reply:
and possibly add some documentation that CMake can report